ubuntudesign/vanilla-docs-theme
# Upgrade to vanilla v1

## Done
- Updated the theme to support Vanilla V1
- Fixed some design review points
### QA
- Load the demo and check it looks ok
- Check small screens
- Check it with Yaili
